src/main/java/use_case/login/LoginInteractor.java

```diff
@@ -39,7 +39,7 @@ public void execute(LoginInputData loginInputData) {
                 loginPresenter.prepareFailView("Incorrect password for \"" + username + "\".");
             }
             else {
-                listInteractor.execute(user);
+                listInteractor.executeOnLogin(user.getAllMediaCollections());
```

Comment:
It's not really feasible to move the method from list interactor to login interactor, since it needs to process the user media collection data. Since interactors are both in the application rules layer, it's still Clean Architecture. 

I did ask my instructor about this and she okayed it, but I get your concern since this creates coupling.